components/org.wso2.carbon.identity.application.authenticator.facebook/src/main/java/org/wso2/carbon/identity/application/authenticator/facebook/FacebookAuthenticator.java

```diff
@@ -750,6 +942,11 @@ public List<Property> getConfigurationProperties() {
         return configProperties;
     }
 
+    @Override
+    public boolean isAPIBasedAuthenticationSupported() {
+        return true;
```

Comment:
missing new line after the method signature

Reply:
Addressed with https://github.com/wso2-extensions/identity-outbound-auth-facebook/pull/64